app/workers/crawler/tabelog.rb

```diff
@@ -0,0 +1,44 @@
+class Crawler::Tabelog < Crawler::Base
+  include Crawler::Searchable
+
+  def initialize
+    super
+    @base_url = 'http://tabelog.com'
+    @site_name = 'tabelog'
+    @redis = Redis.new
+  end
+  def run
+    fetch_places
+    search_at_tabelog
+  end
+
+  def fetch_places
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for fetch_places is too high. [16.19/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)